toolchain/driver/driver.cpp

```diff
@@ -222,7 +225,9 @@ auto Driver::RunDumpSubcommand(DiagnosticConsumer& consumer,
   llvm::LLVMContext llvm_context;
   const std::unique_ptr<llvm::Module> module =
       LowerToLLVM(llvm_context, input_file_name, semantics_ir, vlog_stream_);
+
   CARBON_VLOG() << "*** LowerToLLVM done ***\n";
+
```

Comment:
```suggestion
  CARBON_VLOG() << "*** LowerToLLVM done ***\n";
```

Minor, but the lack of spacing here is deliberately both for consistency with the above, and so that substantial steps are a block.